Firestore/Example/Tests/Integration/API/FIRQueryTests.mm

```diff
@@ -1018,6 +1049,11 @@ - (void)testOrQueriesWithArrayMembership {
 }
 
 - (void)testMultipleInOps {
+  // TODO(orquery): Enable this test against production when possible.
+  XCTSkipIf(![FSTIntegrationTestCase isRunningAgainstEmulator],
+            "Skip this test if running against production because order-by-equality is not "
+            "supported yet.");
```

Comment:
The message is not correct. The queries in this test do not have any OrderBys. Follow [this](https://cs.opensource.google/firebase-sdk/firebase-android-sdk/+/master:firebase-firestore/src/androidTest/java/com/google/firebase/firestore/QueryTest.java;l=1216).

Reply:
The Swift test has the correct message though :)